src/lightning_app/utilities/frontend.py

```diff
@@ -27,19 +32,29 @@ def update_index_file_with_info(ui_root: str, info: AppInfo = None) -> None:
         # revert index.html in case it was modified after creating original.html
         shutil.copyfile(original_file, entry_file)
 
-    if not info:
-        return
+    if info:
+        with original_file.open() as f:
+            original = f.read()
 
-    original = ""
+        with entry_file.open("w") as f:
+            f.write(
+                rewrite_static_with_root_path(_get_updated_content(original=original, root_path=root_path, info=info))
```

Comment:
do we have to pipe this call? Can we `rewrite_static_with_root_path` happens inside `_get_updated_content`.

And probably not even a function, just a section inside `_get_updated_content`
?

Reply:
updated!